pkg/server/routes.go

```diff
@@ -90,4 +90,8 @@ func (s *Server) routes(
 		}
 		api.Handle("/system_intakes", systemIntakesHandler.Handle())
 	}
+
+	s.router.PathPrefix("/").Handler(handlers.CatchAllHandler{
```

Comment:
Can you add an integration test to verify this gets hit if and only if there's no matching route? I'd probably add one for both `/notapath` and `/api/v1/notapath`

Reply:
yep. We were just trying to get a little closer, but that's a good call.